.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,40 @@
+---
+name: golangci-lint
+
+on:
+  pull_request:
+    branches: ['*']
+
+jobs:
+  golangci-lint:
+    name: golangci-lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup go env
```

The verb for setup (none) is set up.
```suggestion
      - name: Set up Go environment 
```